falcon/routing/compiled.py

```diff
@@ -224,13 +224,19 @@ def insert(nodes, path_index=0):
                     return
 
                 if node.conflicts_with(segment):
-                    msg = textwrap.dedent("""
+                    msg = (
+                        textwrap.dedent(
+                            """
                         The URI template for this route is inconsistent or conflicts with another
```

Comment:
Shouldn't the text be moved further left to match the triple quotes?

Reply:
I'll check if it can be tweaked